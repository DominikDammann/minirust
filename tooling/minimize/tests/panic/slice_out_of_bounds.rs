//! Ensures that a normal index operation panics for out of bounds indexes.

#[allow(unconditional_panic)]
fn main() {
    let x = [1, 2_u32];
    let x: &[u32] = &x;
    // Assertion failure in in-bounds check:
    let _y = x[2];
}
