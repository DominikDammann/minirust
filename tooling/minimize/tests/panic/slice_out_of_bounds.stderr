fatal error: Panic: we panicked
